src/gt4py/testing/suites.py

```diff
@@ -379,19 +380,22 @@ def test_generation(self, test, externals_dict):
         implementation = gtscript.stencil(
             backend=test["backend"],
             definition=test["definition"],
-            name=f"{test['suite']}_{test['backend']}_{test['test_id']}",
+            name=f"{test['suite']}_{gt_utils.slugify(test['backend'], valid_symbols='')}_{test['test_id']}",
```

Comment:
I probably did this, but the f-string becomes hard to read

Reply:
Part of the f-string was already there but I made it worse with the `slugify` call. Fixed